### PR TITLE
Add Safari note to CustomElementRegistry.get

### PR DESCRIPTION
**Pull request by tomasdev**

### CODE REVIEW

**Comment by queengooborg** [comment]
Hey there!  I see that the PR description is left blank.  Please fill in the details for your changes, including a brief explanation of what is changed, how you verified the changes are accurate and any supporting evidence you can provide.  Also, please make sure to fix any failing linting.

File: api/CustomElementRegistry.json [hunk 1]
```diff
@@ -155,7 +155,9 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": "10.1"
+              "version_added": "10.1",
+              "partial_implementation": true,
+              "notes": "Supports 'Autonomous custom elements' but not 'Customized built-in elements'. https://bugs.webkit.org/show_bug.cgi?id=182671"
```
Comment (Elchi3) [inline api/CustomElementRegistry.json:160]:
```suggestion
              "notes": "Supports 'Autonomous custom elements' but not 'Customized built-in elements'. See <a href='http://webkit.org/b/182671'>bug 182671 </a>."
```

File: api/CustomElementRegistry.json [hunk 2]
```diff
@@ -155,7 +155,9 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": "10.1"
+              "version_added": "10.1",
+              "partial_implementation": true,
+              "notes": "Supports 'Autonomous custom elements' but not 'Customized built-in elements'. See <a href='http://webkit.org/b/182671'>bug 182671 </a>."
```
Comment (Elchi3) [inline api/CustomElementRegistry.json:160]:
```suggestion
              "notes": "Supports 'Autonomous custom elements' but not 'Customized built-in elements'. See <a href='https://webkit.org/b/182671'>bug 182671</a>."
```

**Review by Elchi3 — APPROVED** [review]
Thank you! 👍

**Pull request merged**